client/app/bundles/course/assessment/submission/components/FileInput.jsx

```diff
@@ -78,7 +78,7 @@ class FileInput extends Component {
     this.setState({ dropzoneActive: false });
     if (!disabled) {
       callback(files);
-      return onChange(files);
+      return onChange(files.length > 0 ? files : null);
```

Comment:
@jeremyyap that time I worked with dropzone, the issue was that I was unable to 'unselect' a 'selected' file, and I can only change it >_< 

Is that what you wish to support?